.github/workflows/frontend-workflow.yml

```diff
@@ -74,16 +85,24 @@ jobs:
         submodules: recursive
         token: ${{ secrets.npm_token }}
 
-    - name: Setup Github as proxy Registry
+    - name: Install node and setup Github as proxy Registry (use npm_version)
       uses: actions/setup-node@v4
+      if: ${{ inputs.use_nvmrc == 'false' }}
       with:
         node-version: '${{ inputs.npm_version }}'
         registry-url: 'https://npm.pkg.github.com/mov-ai'
 
-    - name: Setup .npmrc and install pnpm
+    - name: Install node and setup Github as proxy Registry (use .nvmrc)
+      uses: actions/setup-node@v4
+      if: ${{ inputs.use_nvmrc == 'true' }}
+      with:
+        node-version-file: '.nvmrc'
+        registry-url: 'https://npm.pkg.github.com/mov-ai'
+
+    - name: Install pnpm and setup .npmrc
       if: ${{ inputs.pm == 'pnpm' }}
       run: |
-        npm install -g pnpm
+        npm install -g pnpm@${{ inputs.pnpm_version }}
```

Comment:
what happen if `inputs.use_nvmrc == 'true'` ? `inputs.pnpm_version` is not relevant anymore, isn't ?

Reply:
That just chooses the node version. Not the pnpm version

Reply:
`inputs.use_nvmrc` and `inputs.pnpm_version` are independent. The first makes the action use the file `.nvmrc` to get the node version, the second is the pnpm version